public/app/features/dashboard/components/GenAI/GenAIPanelDescriptionButton.tsx

```diff
@@ -22,10 +22,15 @@ const DESCRIPTION_GENERATION_STANDARD_PROMPT =
 
 export const GenAIPanelDescriptionButton = ({ onGenerate, panel }: GenAIPanelDescriptionButtonProps) => {
   const messages = React.useMemo(() => getMessages(panel), [panel]);
-  const onClick = React.useCallback(() => reportGenerateAIButtonClicked(EventSource.panelDescription), []);
 
   return (
-    <GenAIButton messages={messages} onClick={onClick} onGenerate={onGenerate} loadingText={'Generating description'} />
+    <GenAIButton
+      messages={messages}
+      onGenerate={onGenerate}
+      loadingText={'Generating description'}
+      toggleTipTitle={'Improving your panel description'}
```

Comment:
update tense for more natural feel
```suggestion
      toggleTipTitle={'Improve your panel description'}
```